config/builds.js

```diff
@@ -60,7 +60,8 @@ const builds = buildConfigs
         )
       ],
       clientEntry: path.join(cwd, entry.client || 'src/client.js'),
-      renderEntry: path.join(cwd, entry.render || 'src/render.js'),
+      renderEntry: entry.render ? path.join(cwd, entry.render) : null,
```

Comment:
I did not even know you did not need a config file. Yes, that would be a breaking change. We can make the logic more complex to support this